src/js/gumhandler.js

```diff
@@ -59,8 +61,12 @@ GumHandler.prototype = {
   gotError_: function(error) {
     clearTimeout(this.firstUserCheck_);
     this.gumPendingDialog_.close();
-    this.gumErrorMessage_.innerHTML = error.name;
-    this.gumErrorDialog_.open();
+    if (error.name === 'DevicesNotFoundError') {
+      this.gumNoDeviceDialog_.open();
```

Comment:
It is missing a close of dialog whenever the condition stops being true.

Probably is missing:
 1 - close the gumErrorDialog whenever gumNoDeviceDialog is opened.
 2 - close the gumNoDeviceDialog whenever gumErrorDialog is opened.
 3 - close the gumNoDeviceDialog when gotStream succeeds.